src/plugins/network/README.md

```diff
@@ -14,6 +14,8 @@ This plugin is a check plugin that checks if a key contains a valid ip
 address. It uses the `POSIX.1-2001` interface `getaddrinfo()` in order
 to check if an ip address is valid.
 
+The plugin can also check for valid port numbers and if the set port is free to use.
```

Comment:
The port should actually also passed to getaddrinfo. Better to rewrite the intro.